### PR TITLE
Update Heat_Orchestration_Template_LB.yaml

### PR DESCRIPTION
**Pull request by LeslieLundquist**
Add posting date to heat template

### CODE REVIEW

File: _heat/Heat_Orchestration_Template_LB.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-heat_template_version: 2015-10-15
+heat_template_version: 2015-10-15 (posted 7-5-2016)
```
Comment (omgjlk) [inline _heat/Heat_Orchestration_Template_LB.yaml:1]: I don't think you want to do that, as this is a key that heat reads. The template version indicates a specific syntax to the template. See http://docs.openstack.org/developer/heat/template_guide/hot_spec.html#heat-template-version

You could update the template version with the version supported by Mitaka (`2016-04-08`)

**Comment by LeslieLundquist** [comment]
@ying-tang Could you please update this template to the version Jesse mentions, which is supported by Mitaka?

**Comment by LeslieLundquist** [comment]
@j2sol @ying-tang I had to make a change to open a PR so I could ask you about it, ha ha! Not intending to merge it into the document. Thank you for your help.

**Comment by ying-tang** [comment]
@j2sol Thanks Jesse for your comments. 
@LeslieLundquist Thank you Leslie for noticing this issue. I am opening another PR to update the version. Please see PR#250.

**Pull request closed without merging**